### PR TITLE
ibis.iSDM submission

### PR DESCRIPTION
**Pull request by Martin-Jung**
Quick responses to submission flags in the yaml.  For further explanation and details see the [package website](https://iiasa.github.io/ibis.iSDM/index.html) and in particular the vignettes.

| flag    | Response | Comment |
| -------- | ------- | ------- |
| occ_acquisition | no | Left to the user |
| occ_cleaning  |  yes | The package contains some helper functions for preparing biodiversity data including thinning ([vignette](https://iiasa.github.io/ibis.iSDM/articles/01_data_preparationhelpers.html)). |
| data_integration | yes | Several approaches for integrating information are provided, including joint integration of different data types ([vignette](https://iiasa.github.io/ibis.iSDM/articles/03_integrate_data.html))|
| env_collinearity | yes | Range of options from collinear to `Boruta` application. See [train](https://iiasa.github.io/ibis.iSDM/reference/train.html)|
| env_process | yes | The package contains some helper functions for preparing predictor data including the creation of derivates ([vignette](https://iiasa.github.io/ibis.iSDM/articles/01_data_preparationhelpers.html)). |
| bias | yes | Options available via `add_bias_control()` or `add_offset_bias()` |
| study_region | yes | Key step for any model inference ([vignette](https://iiasa.github.io/ibis.iSDM/articles/02_train_simple_model.html)) |
| backg_sample | yes | Range of options available via `pseudoabs_settings()` |
| data_partitioning | no | Currently not implemented and has to be taken care by the user separately. |
| mod_fit | yes | Easily supported via `train()` ([train](https://iiasa.github.io/ibis.iSDM/reference/train.html)). |
| mod_tuning | yes | Parameter option in `train()` and supported for most engines [train](https://iiasa.github.io/ibis.iSDM/reference/train.html).|
| mod_ensemble | yes | Call `ensemble()` on any number of fitted models |
| mod_stack | yes | Can easily be accomplished through summing outputs from a fitted model (`mod$get_data()`).  |
| mod_evaluate | yes | Call `validate()` on any trained model with independent data. |
| mod_multispecies | no | Not yet supported, specific rank responses might be added in future versions for some engines. |
| mod_mechanistic | yes | Several wrappers for mechanistic species distribution models have been [added](https://iiasa.github.io/ibis.iSDM/articles/05_mechanistic_estimation.html). |
| pred_general | yes | Spatial and Spatial-temporal projections are fully supported through dedicated `scenario()` functions ([vignette](https://iiasa.github.io/ibis.iSDM/articles/04_biodiversity_projections.html)).  |
| pred_extrapolation | yes | Functions for assessing similarity surfaces of covariates (`similarity()`) and limiting factors (`limiting()`) for future projections are supported.  |
| limitingpred_inspect | yes | A range of different functions are supported to summarize model outputs and variable importance, including `summary()`, `effects()`, `partial()` and `spartial()` ([reference](https://iiasa.github.io/ibis.iSDM/reference/index.html)) |
| post_processing | yes | Wide variety of options available to constrain projections ([`add_constraint()`](https://iiasa.github.io/ibis.iSDM/reference/index.html#create-spatial-temporal-projections)) smooth projections (see parameters in `project()`). |
| gui | no | Not planned |
| metadata | yes | A dedicated function that records model input and output parameters has been added (`write_summary()`). |

### CODE REVIEW

File: inst/extdata/packages/ibisiSDM.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: ibis.iSDM
+title: An Integrated model for BiodIversity distribution projectionS
+version: 0.1.0
+author: "Martin Jung, Maximilian Hesselbarth"
+maintainer: Martin Jung <jung@iiasa.ac.at>
+  cran: no
```
Comment (sylvainschmitt) [inline inst/extdata/packages/ibisiSDM.yaml]: This yaml is badly formatted. Please fix.

**Comment by sylvainschmitt** [comment]
@Martin-Jung thanks for your submission. Your metadata check the one on CRAN. I will now seek reviewers for the other fields.

**Comment by sylvainschmitt** [comment]
I'm looking for two reviewers: @jamiemkass, @hannahlowens, @luismurao, @MayaGueguen, @AMBarbosa, @gepinillab, @azizka, @marlonecobos, @babaknaimi, @ecospat, @danlwarren, @sgvignali, @sjevelazco, @damariszurell, @mlammens, @IanOndo, @rvalavi, @AndreaSanchezTapia, @codelab3 are you available? Please have a look at the [reviewing guidelines](https://github.com/sylvainschmitt/sdmverse/blob/main/REVIEWING.md)

**Comment by adamlilith** [comment]
Hey Sylvain,

I’d be happy to do this.  However, I recall I wasn’t sure how to use GitHub in the manner that you’d need for a review.  Would you be able to meet with me online and show me how?  I expect that there will be future reviews required once we get our paper published!

Thanks,
Adam

**Review by hannahlowens — APPROVED** [review]
Looks great! Thanks for the careful work.

**Comment by sylvainschmitt** [comment]
> Looks great! Thanks for the careful work.

Thanks for the review @hannahlowens

**Comment by sylvainschmitt** [comment]
> Hey Sylvain, I’d be happy to do this. However, I recall I wasn’t sure how to use GitHub in the manner that you’d need for a review. Would you be able to meet with me online and show me how? I expect that there will be future reviews required once we get our paper published! 

Hi @adamlilith , sorry I was not available last week but I am now. We can schedule a visio. However, I think I should note the information that helped you and start a collaborative wiki to help authors, reviewers, and editors.

**Comment by jamiemkass** [comment]
Hey folks, this PR was left hanging. Any updates?

**Comment by sylvainschmitt** [comment]
> Hey folks, this PR was left hanging. Any updates?

Sorry @Martin-Jung , my bad I totally forgot. Maybe we should have several editors to avoid that. Based on @hannahlowens I will accept the submission.

**Comment by sylvainschmitt** [comment]
@Martin-Jung  , thanks for the submission of ibis.iSDM. Your contribution can be integrated without further edits. After merging your pull request, you will be added to the repository contributors. Please expect to be contacted for a review request in the future.

**Comment by Martin-Jung** [comment]
Thanks a lot, happy to help :)

**Pull request merged**